### ISSUE TITLE
Measure tool - undo previous measurement with right click of a mouse

### ISSUE THREAD

**Issue by SannaJokela** [oskariorg/oskari-docs#60]
Suggestion to enchancing measurment tool: 

When measuring e.g. long routes, you accidentally click a point that you don't want the measurement to calculate, you could undo the previous measurement with right click of a mouse and then continue measuring. (Suggested through Oskari Joint Development Forum).

**Comment by data-ux** [comment]
Good idea. Undoing the last point could alternatively be done with backspace/delete key. This type of undo is un use for example in Photoshop when drawing selections.

Using right-click would be problematic because most platforms have the context menu tied to it and we wouldn't want to mess with that.
